package.json

```diff
@@ -21,28 +21,28 @@
   "author": "PhraseApp",
   "license": "MIT",
   "devDependencies": {
-    "@types/jest": "^24.0.23",
-    "@types/node": "^9.4.6",
-    "@types/react": "^16.9.17",
-    "@types/react-dom": "^16.9.4",
-    "@types/react-intl": "^2.3.5",
-    "@types/react-test-renderer": "^16.9.1",
-    "jest": "^24.9.0",
+    "@types/jest": "^25.1.2",
+    "@types/node": "^13.7.1",
+    "@types/react": "^16.9.19",
+    "@types/react-dom": "^16.9.5",
+    "@types/react-intl": "^3.0.0",
+    "@types/react-test-renderer": "^16.9.2",
+    "jest": "^25.1.0",
+    "merge": ">=1.2.1",
     "react": "^16.12.0",
     "react-addons-test-utils": "^15.6.2",
     "react-dom": "^16.12.0",
-    "react-intl": "^2.2.3",
+    "react-intl": "^3.0.0",
     "react-test-renderer": "^16.12.0",
-    "ts-jest": "^24.2.0",
-    "ts-loader": "^4.1.0",
-    "typescript": "^3.7.3",
-    "webpack": "^4.41.3",
-    "webpack-cli": "^3.3.10",
-    "merge": ">=1.2.1"
+    "ts-jest": "^25.2.0",
+    "ts-loader": "^6.2.1",
+    "typescript": "^3.7.5",
+    "webpack": "^4.41.6",
+    "webpack-cli": "^3.3.11"
   },
   "peerDependencies": {
-    "react": ">= 15.5.4",
-    "react-intl": ">= 2.2.3 < 3"
+    "react": ">=16.12.0",
```

Comment:
Isn't this too restrict? Is there any issue with older releases?

Reply:
You're right. Just checked and `react-intl` has peerDependency of `react` with version `^16.3.0`, so we should go with the same.
Another question would be - do we need to specify `react` peerDependency at all, when we have `react-intl` which should always require `react` itself?

Reply:
I don't really have a strong opinion on this. Remove it if this feels right to you :)

Reply:
We have the `react` required in our codebase, so maybe it's better to leave it here, I'll just upgrade the version number

Reply:
Done!